packages/frontend/app/services/channels/-utils/channel-saver.ts

```diff
@@ -0,0 +1,75 @@
+import Channel from 'emberclear/models/channel';
+import Vote from 'emberclear/models/vote';
+import VoteChain from 'emberclear/models/vote-chain';
+import ChannelContextChain from 'emberclear/models/channel-context-chain';
+import Identity from 'emberclear/models/identity';
+
+export async function saveChannel(channel: Channel) {
+  console.error('before saving channel admin');
```

Comment:
remember to remove all these consoles ;)

Reply:
Will do. We are still in the process of debugging the tests, so once we remodel the models, we will continue

Reply:
Addressed in ae248b3